doc/api/http.md

```diff
@@ -907,9 +907,7 @@ added: v0.4.0
 * `name` {String}
 * Returns: {String}
 
-Reads out a header that's already been queued but not sent to the client.  Note
-that the name is case insensitive.  This can only be called before headers get
-implicitly flushed.
+Reads out a header that's already been queued but not sent to the client.  Note that the name is case insensitive
```

Comment:
Could you keep the first two sentences wrapped to <80chars? The diff should look like:

```diff
-that the name is case insensitive.  This can only be called before headers get
-implicitly flushed.
+that the name is case insensitive.
```